docs/windows.md

```diff
@@ -171,9 +173,9 @@ the address for kubelet. It should look like:
 KUBELET_KUBEADM_ARGS="--cgroup-driver= --network-plugin=cni --pod-infra-container-image=k8s.gcr.io/pause:3.1 --node-ip=$NODE_IP"```
 ```
 
-4. Prepare network adapter for kubernetes service
+4. Prepare network adapter for kube-proxy
 
-kube-proxy need a network adapter to configure kubernetes service IPs and use
+kube-proxy needs a network adapter to configure Kubernetes Service IPs and uses
 the adapter for network proxy. Use following script to create the network
```

Comment:
suggestion: s/for network proxy/for proxying connections to Services

I think it makes things clearer